### PR TITLE
~ escaping function argument in LESS is considered to be invalid

### PR DESCRIPTION
**Pull request by varya**
Refers to issue #40
https://github.com/tonyganch/gonzales-pe/issues/40

### CODE REVIEW

**Comment by tonyganch** [comment]
Could you add a test, please?

**Comment by varya** [comment]
Added

File: lib/less/rules.js [hunk 1]
```diff
@@ -1473,7 +1474,8 @@ module.exports = (function() {
         pos++;
 
         while (pos < tokensLength && tokens[pos].type !== TokenType.RightParenthesis) {
-            if (checkDeclaration(pos)) x.push(getDeclaration());
+            if (checkEscapedString(pos)) x.push(getEscapedString());
```
Comment (tonyganch) [inline lib/less/rules.js]: Кажется, это нужно перенести в [getArgument](https://github.com/varya/gonzales-pe/blob/features/escaping-tildas-in-less/lib/less/rules.js#L233).

**Comment by varya** [comment]
Moved checking under "getArgument" method.

**Comment by tonyganch** [comment]
Thank you!
Do you need version 3.0.0-11 or can `dev` branch stay unpublished yet?

**Comment by varya** [comment]
Yeas, I was about to ask you to release 3.0.0-11. Could you please?

**Comment by tonyganch** [comment]
Done.

**Pull request merged**